lib/internal/test_runner/reporter/dot.js

```diff
@@ -23,6 +29,14 @@ module.exports = async function* dot(source) {
     }
   }
   yield '\n';
+
+  if (failedTests.length > 0) {
+    yield `${colors.red}${failedTestSymbol} failing tests:${colors.white}\n`;
```

Comment:
We might want to use `util.styleText`, though I am not sure we want to, as it could be affected from user-land

Reply:
since the issue mentioned spec reporter, I tried to match it to how it's done in spec.js. If it is required to be changed, should it only be done for dot reporter or for spec reporter as well?

Reply:
The `util.styleText` was added after these reporters 🙂 
Anyway, not sure we want to use it here. @MoLow @cjihrig WDYT?

Reply:
It might be worth to change it globally, but I think that’s an issue for a different PR.